edgePy/_DGEList.py

```diff
@@ -116,10 +116,20 @@ def samples(self) -> np.array:
 
     @samples.setter
     def samples(self, samples: Optional[np.ndarray]) -> None:
-        # TODO: Validate samples here
-        # - Samples same length as ncol(self.counts) if defined
+        """Validate setting ``DGEList.samples`` for the illegal conditions:
+
+            * Must be the same length as the columns in counts`
+
+        Args:
+            samples: 1D string array representing identifiers of count columns
+
+        """
```

Comment:
These docstrings, although spartan for methods like these will look really good when compiled by Sphinx!

Reply:
Oh yeah! I should work on that. Will do